Lesson15/Merve2/script.js

```diff
@@ -0,0 +1,28 @@
+// 4. Use the `toLowerCase` method to convert a string to lowercase and print it to the console.
+const stringExercise4 =
+	'FUNCTIONAL PROGRAMMING EMPHASIZES PURE FUNCTIONS AND IMMUTABLE DATA.';
+
+console.log('Exercise 4: ');
+console.log(stringExercise4.toLowerCase());
+console.log('------------------------');
+
+// 6. Use the `lastIndexOf` method to find the last index of of "l" character in the string stringExercise5 
+//and print it to the console.
+
+console.log('Exercise6: ');
+console.log(stringExercise4.length);
+console.log(stringExercise4.toLowerCase().lastIndexOf('l'));
+console.log('------------------------');
+
+// 15. Remove extra spaces from the string stringExercise15 and print the result to the console.
+const stringExercise15 = '    Hello world    ';
+
+console.log('Exercise15: ');
+console.log(stringExercise15.trim());
+console.log('------------------------');
+// 16. Extract the price and currency from a string below using "slice" method and print it to console.
+const planPriceString = 'Premium plan - 9.99 USD/month.'; // Should return "9.99 USD"
+
+console.log('Exercise16: ');
+console.log(planPriceString.slice(15, 23));
```

Comment:
In real life, it will often not be possible to use hardcoded indexes like 15 and 23 so instead, you'd need to find them based on other things in the string. E.g. there's a "- " just before the price and "/" just after it.
So try to re-do this exercise by:
- find start index by finding the position of the "- " characters
- find end index by finding the position of the "/" character
- use that indexes in the slice function